uber/custom_tags.py

```diff
@@ -503,12 +503,14 @@ def __init__(self, label, takedown, amount_extra='0'):
         self.takedown, self.amount_extra = Variable(takedown), Variable(amount_extra)
 
     def _notice(self, label, takedown, amount_extra):
```

Comment:
This implementation relies on ``c.GROUP_TAKEDOWN`` being on a different date from other takedown dates, which will not necessarily be true, even if it happens to be the case for the current MAG events.  We should probably instead just pass in the ``discount`` field or otherwise indicate whether or not this is a group registration.